pykrige/ok.py

```diff
@@ -606,74 +553,95 @@ def _exec_loop_moving_window(self, a_all, bd_all, mask, bd_idx):
+        'exact'. This means that, when a specified coordinate for interpolation
+        is exactly the same as one of the data points, the variogram evaluated
+        at the point is forced to be zero. Also, the diagonal of the kriging
+        matrix is also always forced to be zero. In forcing the variogram
+        evaluated at data points to be zero, we are effectively saying that
+        there is no variance at that point (no uncertainty,
         so the value is 'exact').
 
-        In the future, the code may include an extra 'exact_values' boolean flag that can be
-        adjusted to specify whether to treat the measurements as 'exact'. Setting the flag
-        to false would indicate that the variogram should not be forced to be zero at zero distance
-        (i.e., when evaluated at data points). Instead, the uncertainty in the point will be
-        equal to the nugget. This would mean that the diagonal of the kriging matrix would be set to
+        In the future, the code may include an extra 'exact_values' boolean
+        flag that can be adjusted to specify whether to treat the measurements
+        as 'exact'. Setting the flag to false would indicate that the
+        variogram should not be forced to be zero at zero distance
+        (i.e., when evaluated at data points). Instead, the uncertainty in
+        the point will be equal to the nugget. This would mean that the
+        diagonal of the kriging matrix would be set to
         the nugget instead of to zero.
 
-        Inputs:
-            style (string): Specifies how to treat input kriging points.
-                Specifying 'grid' treats xpoints and ypoints as two arrays of
-                x and y coordinates that define a rectangular grid.
-                Specifying 'points' treats xpoints and ypoints as two arrays
-                that provide coordinate pairs at which to solve the kriging system.
-                Specifying 'masked' treats xpoints and ypoints as two arrays of
-                x and y coordinates that define a rectangular grid and uses mask
-                to only evaluate specific points in the grid.
-            xpoints (array-like, dim N): If style is specific as 'grid' or 'masked',
+        Parameters
+        ----------
+            style : str
+                Specifies how to treat input kriging points. Specifying 'grid'
+                treats xpoints and ypoints as two arrays of x and y coordinates
+                that define a rectangular grid. Specifying 'points' treats
+                xpoints and ypoints as two arrays that provide coordinate pairs
+                at which to solve the kriging system. Specifying 'masked'
+                treats xpoints and ypoints as two arrays of x and y coordinates
+                that define a rectangular grid and uses mask to only evaluate
+                specific points in the grid.
+            xpoints : array_like, dim N

```

Comment:
In all the places in the docstrings where we use `dim` it might be better to replace it by `shape` (the dim is always 1, 2  or 3 here).

Reply:
Will do